async-jms/apps/async-jms/src/main/java/com/redhat/training/model/Order.java

```diff
@@ -0,0 +1,127 @@
+package com.redhat.training.model;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+public class Order implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+	
+	@JsonProperty("ID")
+	private Integer id;
+	@JsonIgnore
+	private Date orderDate = new Date();
+	@JsonProperty("Discount")
+	private BigDecimal discount;
+	@JsonProperty("Delivered")
+	private Boolean delivered=false;
+	@JsonProperty("Desc")
+	private String description;
```

Comment:
Please add empty lines between the different attributes to make the code more clear